frontend/components/PaginatedList/PaginatedList.tsx

```diff
@@ -0,0 +1,279 @@
+import React, {
+  useState,
+  useEffect,
+  useImperativeHandle,
+  forwardRef,
+  Ref,
+} from "react";
+import classnames from "classnames";
+import { ReactElement } from "react-markdown/lib/react-markdown";
+import Checkbox from "components/forms/fields/Checkbox";
+import Spinner from "components/Spinner";
+import TooltipTruncatedText from "components/TooltipTruncatedText";
+// @ts-ignore
```

Comment:
No longer need the `// @ts-ignore` since the new Pagination is typescript file <3